Tools/MGCB/ConsoleLogger.cs

```diff
@@ -11,13 +11,13 @@ public class ConsoleLogger : ContentBuildLogger
     {
         public override void LogMessage(string message, params object[] messageArgs)
         {
-			Console.WriteLine(message, messageArgs);
+			Console.WriteLine(IndentString + message, messageArgs);
```

Comment:
> ok, so because most (all?) content exceptions are caught outside the `_manager.BuildContent()`

Oh... I see what you are saying.  The capture and logging of the content exception is handled above you, so the indent is already lost.

This seems like it would require that `BuildContent()` internally be responsible for at least logging exceptions.  We can still re-throw and let the caller deal with the rest.